SR2023/sr2023-markers

```diff
@@ -9,5 +9,5 @@ rm -r markers
 zoloto marker-pdfs APRILTAG_36H11 markers 200 --description-format 'Student Robotics {marker_type} - #{marker_id}' --range 0-27 --page-size A3
 rename .pdf _a3.pdf markers/*
 zoloto marker-pdfs APRILTAG_36H11 markers 200 --description-format 'Student Robotics {marker_type} - #{marker_id}' --range 0-27 --page-size A3 --force-a4
-zoloto marker-pdfs APRILTAG_36H11 markers 80 --description-format 'Student Robotics {marker_type} - #{marker_id}' --range 27-99 --page-size A4
+zoloto marker-pdfs APRILTAG_36H11 markers 80 --description-format 'Student Robotics {marker_type} - #{marker_id}' --range 28-100 --page-size A4
```

Comment:
Since range is exclusive at the upper bound 100 is not a marker: https://github.com/srobo/sr-robot3/blob/main/sr/robot3/game.py#L11


```suggestion
zoloto marker-pdfs APRILTAG_36H11 markers 80 --description-format 'Student Robotics {marker_type} - #{marker_id}' --range 28-99 --page-size A4
```

Reply:
Ah, that looks like a bug in the robot API code -- 100 _is_ intended to be a token marker: https://studentrobotics.org/docs/resources/2023/rulebook.html#marker-specification
Probably my fault as I think I made that API code change.

Reply:
https://github.com/srobo/sr-robot3/pull/87 aims to fix that.